.husky/pre-commit

```diff
@@ -0,0 +1,11 @@
+echo "🔥 Husky pre-commit running..."
```

Comment:
your huski is not working, I tried to commit the file which contains eslint error and it was successfully committed